utils/entitlements.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // ProjectEntitlementsByBundleID ...
 func ProjectEntitlementsByBundleID(pth, schemeName, configurationName string) (map[string]plistutil.PlistData, error) {
-	var scheme xcscheme.Scheme
+	var scheme *xcscheme.Scheme
```

Comment:
pls check this comment: https://github.com/bitrise-io/xcode-project/pull/27#issue-250892062
the new version of the xcode-proj package makes this block much shorter

Reply:
Updated code.